.travis.yml

```diff
@@ -1,16 +1,33 @@
 sudo: enabled
-dist: xenial
 language: r
 warnings_are_errors: true
 
-# Only allow the lintr-bot to comment on the release build (this avoids duplicate comments)
+# Use Xenial whenever running in Linux (ignored for OS X)
+dist: xenial
+
+# Disable lintr-bot comments by default
+env:
+  - LINTR_COMMENT_BOT=false
+
+# Construct the matrix of OS and R versions
+os:
+  - linux
+  - osx
+r:
+  - oldrel
+  - release
+  - devel
+
+# Explicitly enable the lintr-bot for one build (this avoids duplicate comments)
 matrix:
-  include:
-    - r: oldrel
-      env: LINTR_COMMENT_BOT=false
-    - r: release
-    - r: devel
+  exclude:
+    - os: linux
```

Comment:
Why do we keep the `exclude` in lines 23-26 now that `LINTR_COMMENT_BOT` is disabled by default?

Reply:
Travis treats builds with different environment variables as different builds. We therefore remove the one we don't want in order to replace it with the one we do. I've tried using conditions to change this variable, but I don't think it works - will revert.